### DISCUSSION TITLE
Uninstalling on Windows 11

### DISCUSSION THREAD

**Discussion by BilleBarrett** [Open-Shell/Open-Shell-Menu#929] [General]
The client wanted Open-Shell uninstalled and after I did it the Windows 11 Start does not work
I tried on another windows 11 computer and it did the same
In both cases, I had to reinstall Open-Shell Menu
Any suggestions as to why this is happening

**Comment by YesTrustMe** [comment]
It hasn't been updated in quite some time and W11 isn't listed as being compatible. There is a pre-release version. Perhaps you can try that?
https://github.com/Open-Shell/Open-Shell-Menu/releases/tag/4.4.169-beta
